scripts/compile_coq.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh -e
+
+help() {
+  echo "./compile-coq <compiler path> <path to directory> [--opam opam] [-c|--compile-opam] [-r|--remove]"
+  echo "[compiler path]     is the path to the ocaml compiler"
+  echo "[path to directory] is the path to the drectory where eveything will be build." 
+  echo "                    This script considers that it is free to do anything from"
+  echo "                    this directory, including deleting it."
+  echo "                    Omitting this entry will result in using a temp dir."
+  echo "[-c|--compile-opam] Compile and use the latest version of opam."
+  echo "[--opam opam]       Use a specific version of opam."
```

Comment:
Rather than a "version" of opam (personally, for me "version" would refer to the version number), this arguments takes the opam command/binary to execute, no ?

Reply:
Done